content/applications/hr/employees/new_employee.rst

```diff
@@ -103,313 +119,389 @@ following information for each entry.
+
+  Use the breadcrumb links to navigate back to the new employee form when done.
+
+  .. important::
+     The users that appear in the drop-down menu for the :guilabel:`Approvers` section **must** have
+     *Administrator* rights set for the corresponding human resources role.
+
+     To check who has these rights, go to :menuselection:`Settings app --> Users --> → Manage
+     Users`. Then, click on an employee, and check the :guilabel:`Human Resources` section of the
+     :guilabel:`Access Rights` tab.
+
+     - In order for the user to appear as an approver for :guilabel:`Expenses`, they **must** have
+       either :guilabel:`Team Approver`, :guilabel:`All Approver`, or :guilabel:`Administrator` set
+       for the :guilabel:`Expenses` role.
+     - In order for the user to appear as an approver for :guilabel:`Time Off`, they **must** have
+       either :guilabel:`Officer:Manage all Requests` or :guilabel:`Administrator` set for the
+       :guilabel:`Time Off` role.
+     - In order for the user to appear as an approver for :guilabel:`Timesheets`, they **must** have
+       either :guilabel:`Manager`, :guilabel:`Officer:Manage all contracts`, or
+       :guilabel:`Administrator` set for the :guilabel:`Payroll` role.
+
+- :guilabel:`Remote Work`: use the drop-down menu to select the default location the employee works
+  from each day of the week. The default options are :guilabel:`Home`, :guilabel:`Office`, or
+  :guilabel:`Other`.
+
+  A new location can be typed into the field, then click either :guilabel:`Create (new location)` to
+  add the location, or :guilabel:`Create and edit...` to add the new location and edit the form.
+
+  After edits are done, click :guilabel:`Save & Close`, and the new location is added, and populates
+  the field.
+
+  Leave the field blank (:guilabel:`Unspecified`) for non-working days like Saturday and Sunday.
 
   .. note::
      It is also possible to add or modify work locations by navigating to :menuselection:`Employees
-     app --> Configuration --> Employee: Work Locations`. Click :guilabel:`New` to create a new
-     location, then enter the details on the form. To modify a location, click on an existing
+     app --> Configuration --> Employee: Work Locations`. To modify a location, click on an existing
      location, then make any changes on the form.
 
-- :guilabel:`Schedule`: Select the :guilabel:`Working Hours` and :guilabel:`Timezone` for the
-  employee. The :guilabel:`Internal Link` arrow opens up a detailed view of the specific daily
-  working hours. Working hours can be modified or deleted here.
+     Click :guilabel:`New` to create a new location, then enter the following information on the
+     form. All fields are **required**.
+
+     - :guilabel:`Work Location`: enter the name for the location. This can be as general or
+       as specific, as needed, such as `Home` or `Building 1, Second Floor`, respectfully.
```

Comment:
Early break
```suggestion
     - :guilabel:`Work Location`: enter the name for the location. This can be as general or as
       specific, as needed, such as `Home` or `Building 1, Second Floor`, respectfully.
```